core/src/apps/bitcoin/sign_tx/helpers.py

```diff
@@ -350,7 +350,11 @@ def sanitize_tx_prev_input(txi: PrevInput, coin: CoinInfo) -> PrevInput:
 def sanitize_tx_output(txo: TxOutput, coin: CoinInfo) -> TxOutput:
     if txo.multisig and txo.script_type not in common.MULTISIG_OUTPUT_SCRIPT_TYPES:
         raise wire.DataError("Multisig field provided but not expected.")
-    if txo.address_n and txo.script_type not in common.CHANGE_OUTPUT_SCRIPT_TYPES:
+    if (
+        txo.address_n
+        and txo.script_type not in common.CHANGE_OUTPUT_SCRIPT_TYPES
+        and txo.script_type != OutputScriptType.SSTXCOMMITMENTOWNED
+    ):
```

Comment:
Should also add a coin.decred check here?

Reply:
Sure